xtask/src/bin_version.rs

```diff
@@ -9,5 +9,6 @@ pub const GRCOV: CargoPackage = CargoPackage::new("grcov", "0.8.20");
 pub const WASM_PACK: CargoPackage = CargoPackage::new("wasm-pack", "0.13.1");
 pub const TYPOS_CLI: CargoPackage = CargoPackage::new("typos-cli", "1.29.5").with_binary_name("typos");
 pub const DIPLOMAT_TOOL: CargoPackage = CargoPackage::new("diplomat-tool", "0.7.1");
+pub const NIGHTLY_TOOLCHAIN: &str = "nightly-2025-07-28";
 
 pub const WABT_VERSION: &str = "1.0.36";
```

Comment:
```suggestion

pub const WABT_VERSION: &str = "1.0.36";
pub const NIGHTLY_TOOLCHAIN: &str = "nightly-2025-07-28";
```

Reply:
I will update it. First, I want to make sure CI completes successfully with the fix

Reply:
Fuzzing passed successfully 😎

Reply:
Done